src/components/selector-dropdown/SelectorDropdown.scss

```diff
@@ -13,7 +13,7 @@
         @include bdl-Overlay-container;
     }
 
-    .overlay-wrapper,
```

Comment:
removing this isn't safe. Admin Console has some usages like `.SelectorDropdown .overlay-wrapper`.

you would need to audit and fix those in the parent apps if this is merged

Reply:
thanks for catching that. I added it back in

Reply:
whoops, I commented this on the wrong line. I meant the `${OVERLAY_WRAPPER_CLASS} is-visible` part needs to stay. so it'll be something like:

```
<div className={`SelectorDropdown-overlay ${OVERLAY_WRAPPER_CLASS} is-visible`}>
```

but the line in this file could actually be removed. i.e. replace `.overlay-wrapper` with `.SelectorDropdown-overlay` here

Reply:
this is because the `overlay-wrapper` is being targeted in an Admin Console stylesheet, so we can't remove the classname from the element itself

Reply:
ah that makes sense